functions/index.js

```diff
@@ -469,6 +469,44 @@ module.exports.reloadsheetdata = functions.https.onRequest(async (req, res) => {
   res.status(200).send(html_snippets);
 });
 
+async function updateSheetWithGeocodes(country) {
+  const client = await getAuthorizedClient();
+  let data = {}
+
+  // Open relevant sheet
+  const sheets = google.sheets('v4');
+  const request = {
+    spreadsheetId: SHEETS[country],
+    range: 'Form Responses 1'
```

Comment:
I think now it's safe to just read from Combined always?

Reply:
FYI, Poland appears to be in a half-baked state.  Fortunately it's not in the list of sheets in this script.